### PR TITLE
Bugfix/unity5.4 support

### PR DESCRIPTION
**Pull request by drhaynes**
This PR removes the deprecation warnings present when using TouchScript in a Unity 5.4 project.

### CODE REVIEW

File: Source/Assets/TouchScript/Scripts/Devices/Display/GenericDisplayDevice.cs [hunk 1]
```diff
@@ -46,10 +46,12 @@ protected override void OnEnable()
                     case RuntimePlatform.OSXEditor:
                     case RuntimePlatform.OSXDashboardPlayer:
                     case RuntimePlatform.OSXPlayer:
+#if !UNITY_5_4
+                    case RuntimePlatform.WindowsWebPlayer:
                     case RuntimePlatform.OSXWebPlayer:
+#endif
```
Comment (drhaynes) [inline Source/Assets/TouchScript/Scripts/Devices/Display/GenericDisplayDevice.cs]: Web player has been removed from 5.4

**Comment by valyard** [comment]
Thanks.
Keep in mind that there are also these defines:
#UNITY_5_3_OR_NEWER
#UNITY_5_4_OR_NEWER
#UNITY_5_5_OR_NEWER

**Comment by drhaynes** [comment]
Ah, thanks. Have updated to use those.

**Comment by valyard** [comment]
Merged to develop branch.

**Pull request closed without merging**